apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1961,6 +1965,368 @@ static void libc_stdio_setvbuf_tc(void)
+* @apicovered       lib_nullinstream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_nullinstream_tc(void)
+{
+	FAR struct lib_instream_s *stream;
+	struct lib_instream_s nullinstream;
+
+	lib_nullinstream(&nullinstream);
+	stream = &nullinstream;
+	TC_ASSERT_EQ("lib_nullinstream", nullinstream.nget, 0);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         libc_stdio_nulloutstream_tc
+* @brief            Initializes a NULL stream. The initialized stream will write all data to the bit-bucket
+* @scenario         Initializes a NULL stream. The initialized stream will write all data to the bit-bucket
+* @apicovered       lib_nulloutstream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_nulloutstream_tc(void)
+{
+	FAR struct lib_outstream_s *stream;
+	struct lib_outstream_s nulloutstream;
+
+	lib_nulloutstream(&nulloutstream);
+	stream = &nulloutstream;
+	TC_ASSERT_EQ("lib_nulloutstream", nulloutstream.nput, 0);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         libc_stdio_rawinstream_tc
+* @brief            Initializes a stream for use with a file descriptor
+* @scenario         Initializes a stream for use with a file descriptor
+* @apicovered       lib_rawinstream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_rawinstream_tc(void)
+{
+	int fd;
+	char *filename = VFS_FILE_PATH;
+
+	struct lib_rawinstream_s rawinstream;
+
+	fd = open(filename, O_RDONLY);
+	TC_ASSERT_NEQ("open", fd, OK);
+
+	lib_rawinstream((FAR struct lib_rawinstream_s *)&rawinstream, fd);
+	close(fd);
+	TC_ASSERT_EQ("lib_rawinstream", rawinstream.fd, fd);
+	TC_ASSERT_EQ("lib_rawinstream", rawinstream.public.nget, 0);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         libc_stdio_rawoutstream_tc
+* @brief            Initializes a stream for use with a file descriptor
+* @scenario         Initializes a stream for use with a file descriptor
+* @apicovered       lib_rawoutstream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_rawoutstream_tc(void)
+{
+	int fd;
+	char *filename = VFS_FILE_PATH;
+
+	struct lib_rawoutstream_s rawoutstream;
+
+	fd = open(filename, O_RDONLY);
+	TC_ASSERT_NEQ("open", fd, OK);
+
+	lib_rawoutstream((FAR struct lib_rawoutstream_s *)&rawoutstream, fd);
+	close(fd);
+	TC_ASSERT_EQ("lib_rawoutstream", rawoutstream.fd, fd);
+	TC_ASSERT_EQ("lib_rawoutstream", rawoutstream.public.nput, 0);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         libc_stdio_rawsistream_tc
+* @brief            Initializes a stream for use with a file descriptor
+* @scenario         Initializes a stream for use with a file descriptor
+* @apicovered       lib_rawsistream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_rawsistream_tc(void)
+{
+	int fd;
+	char *filename = VFS_FILE_PATH;
+
+	struct lib_rawsistream_s rawsistream;
+
+	fd = open(filename, O_RDONLY);
+	TC_ASSERT_NEQ("open", fd, OK);
```

Comment:
This is wrong because you should check whether fd is not greater than equal to 0.
So It should be TC_ASSERT_GEQ("open", fd, 0);

Reply:
done